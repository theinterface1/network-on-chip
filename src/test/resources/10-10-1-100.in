         1 (  5,  5) (  7,  5)          1
         2 (  8,  0) (  2,  5)          3
         3 (  5,  4) (  1,  0)          3
         4 (  9,  7) (  5,  3)          4
         5 (  6,  1) (  4,  0)          6
         6 (  7,  3) (  6,  8)          8
         7 (  9,  9) (  3,  2)          9
         8 (  8,  7) (  4,  1)         11
         9 (  5,  3) (  7,  0)         11
        10 (  9,  1) (  5,  1)         12
        11 (  3,  7) (  1,  4)         14
        12 (  4,  1) (  6,  5)         16
        13 (  1,  6) (  2,  3)         16
        14 (  6,  2) (  9,  7)         17
        15 (  8,  9) (  6,  2)         18
        16 (  1,  8) (  6,  5)         19
        17 (  3,  3) (  3,  8)         21
        18 (  8,  8) (  9,  3)         22
        19 (  1,  5) (  8,  4)         22
        20 (  4,  6) (  0,  2)         23
        21 (  5,  0) (  9,  5)         24
        22 (  2,  2) (  6,  1)         26
        23 (  7,  4) (  8,  8)         27
        24 (  7,  0) (  7,  7)         28
        25 (  1,  4) (  0,  5)         29
        26 (  9,  3) (  8,  7)         31
        27 (  0,  5) (  0,  1)         32
        28 (  8,  3) (  1,  2)         34
        29 (  5,  8) (  8,  7)         35
        30 (  3,  6) (  8,  3)         36
        31 (  9,  5) (  0,  1)         38
        32 (  4,  1) (  0,  4)         39
        33 (  9,  4) (  9,  4)         41
        34 (  0,  8) (  4,  3)         42
        35 (  5,  3) (  8,  3)         43
        36 (  6,  3) (  0,  7)         44
        37 (  1,  4) (  5,  2)         46
        38 (  6,  6) (  7,  7)         46
        39 (  6,  2) (  8,  1)         47
        40 (  1,  2) (  1,  9)         49
        41 (  6,  2) (  4,  6)         50
        42 (  8,  0) (  5,  7)         50
        43 (  6,  5) (  2,  3)         52
        44 (  6,  5) (  0,  1)         54
        45 (  0,  9) (  1,  1)         55
        46 (  3,  8) (  9,  0)         56
        47 (  0,  0) (  2,  3)         57
        48 (  2,  0) (  2,  6)         58
        49 (  9,  8) (  7,  1)         60
        50 (  7,  5) (  9,  3)         61
        51 (  9,  8) (  8,  9)         61
        52 (  6,  9) (  4,  4)         63
        53 (  6,  7) (  8,  7)         64
        54 (  7,  0) (  8,  6)         66
        55 (  0,  0) (  6,  7)         66
        56 (  4,  6) (  3,  5)         66
        57 (  7,  6) (  3,  1)         67
        58 (  2,  7) (  7,  2)         69
        59 (  9,  5) (  4,  1)         69
        60 (  2,  3) (  4,  2)         71
        61 (  7,  7) (  9,  0)         72
        62 (  7,  2) (  5,  9)         73
        63 (  5,  0) (  3,  5)         74
        64 (  7,  1) (  1,  8)         74
        65 (  1,  6) (  9,  5)         76
        66 (  0,  5) (  6,  9)         78
        67 (  9,  7) (  6,  3)         79
        68 (  3,  9) (  4,  2)         81
        69 (  9,  6) (  0,  3)         82
        70 (  0,  3) (  3,  3)         82
        71 (  7,  0) (  1,  4)         84
        72 (  2,  4) (  0,  1)         85
        73 (  6,  1) (  3,  2)         86
        74 (  0,  8) (  7,  1)         88
        75 (  5,  2) (  1,  7)         88
        76 (  0,  8) (  5,  9)         88
        77 (  3,  2) (  6,  4)         90
        78 (  9,  8) (  1,  7)         91
        79 (  9,  3) (  5,  6)         92
        80 (  3,  4) (  5,  1)         92
        81 (  6,  9) (  7,  5)         92
        82 (  2,  4) (  1,  8)         93
        83 (  3,  8) (  3,  6)         94
        84 (  4,  1) (  2,  4)         94
        85 (  1,  1) (  5,  0)         95
        86 (  0,  4) (  7,  4)         96
        87 (  4,  7) (  6,  3)         96
        88 (  5,  1) (  4,  5)         97
        89 (  3,  4) (  6,  2)         98
        90 (  4,  0) (  1,  0)        100
        91 (  4,  4) (  1,  9)        100
        92 (  6,  6) (  3,  2)        101
        93 (  4,  3) (  4,  7)        102
        94 (  4,  5) (  3,  1)        103
        95 (  0,  4) (  6,  2)        103
        96 (  3,  2) (  2,  1)        104
        97 (  9,  8) (  5,  7)        105
        98 (  0,  6) (  2,  5)        105
        99 (  4,  4) (  4,  4)        107
       100 (  2,  8) (  0,  5)        107
