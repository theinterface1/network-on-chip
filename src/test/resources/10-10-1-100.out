msg 1 sent by (5, 5) at 1, delivered to (7, 5) at 4
msg 2 sent by (8, 0) at 3, delivered to (2, 5) at 13
msg 3 sent by (5, 4) at 3, delivered to (1, 0) at 16
msg 4 sent by (9, 7) at 4, delivered to (5, 3) at 17
msg 5 sent by (6, 1) at 6, delivered to (4, 0) at 24
msg 6 sent by (7, 3) at 8, delivered to (6, 8) at 23
msg 7 sent by (9, 9) at 9, delivered to (3, 2) at 17
msg 8 sent by (8, 7) at 11, delivered to (4, 1) at 22
msg 9 sent by (5, 3) at 11, delivered to (7, 0) at 21
msg 10 sent by (9, 1) at 12, delivered to (5, 1) at 19
msg 11 sent by (3, 7) at 14, delivered to (1, 4) at 30
msg 12 sent by (4, 1) at 16, delivered to (6, 5) at 23
msg 13 sent by (1, 6) at 16, delivered to (2, 3) at 25
msg 14 sent by (6, 2) at 17, delivered to (9, 7) at 26
msg 15 sent by (8, 9) at 18, delivered to (6, 2) at 30
msg 16 sent by (1, 8) at 19, delivered to (6, 5) at 32
msg 17 sent by (3, 3) at 21, delivered to (3, 8) at 27
msg 18 sent by (8, 8) at 22, delivered to (9, 3) at 29
msg 19 sent by (1, 5) at 22, delivered to (8, 4) at 39
msg 20 sent by (4, 6) at 23, delivered to (0, 2) at 36
msg 21 sent by (5, 0) at 24, delivered to (9, 5) at 34
msg 22 sent by (2, 2) at 26, delivered to (6, 1) at 40
msg 23 sent by (7, 4) at 27, delivered to (8, 8) at 33
msg 24 sent by (7, 0) at 28, delivered to (7, 7) at 36
msg 25 sent by (1, 4) at 29, delivered to (0, 5) at 40
msg 26 sent by (9, 3) at 31, delivered to (8, 7) at 45
msg 27 sent by (0, 5) at 32, delivered to (0, 1) at 39
msg 28 sent by (8, 3) at 34, delivered to (1, 2) at 47
msg 29 sent by (5, 8) at 35, delivered to (8, 7) at 48
msg 30 sent by (3, 6) at 36, delivered to (8, 3) at 49
msg 31 sent by (9, 5) at 38, delivered to (0, 1) at 46
msg 32 sent by (4, 1) at 39, delivered to (0, 4) at 49
msg 33 sent by (9, 4) at 41, delivered to (9, 4) at 42
msg 34 sent by (0, 8) at 42, delivered to (4, 3) at 52
msg 35 sent by (5, 3) at 43, delivered to (8, 3) at 47
msg 36 sent by (6, 3) at 44, delivered to (0, 7) at 53
msg 37 sent by (1, 4) at 46, delivered to (5, 2) at 59
msg 38 sent by (6, 6) at 46, delivered to (7, 7) at 49
msg 39 sent by (6, 2) at 47, delivered to (8, 1) at 59
msg 40 sent by (1, 2) at 49, delivered to (1, 9) at 57
msg 41 sent by (6, 2) at 50, delivered to (4, 6) at 63
msg 42 sent by (8, 0) at 50, delivered to (5, 7) at 65
msg 43 sent by (6, 5) at 52, delivered to (2, 3) at 67
msg 44 sent by (6, 5) at 54, delivered to (0, 1) at 65
msg 45 sent by (0, 9) at 55, delivered to (1, 1) at 59
msg 46 sent by (3, 8) at 56, delivered to (9, 0) at 65
msg 47 sent by (0, 0) at 57, delivered to (2, 3) at 63
msg 48 sent by (2, 0) at 58, delivered to (2, 6) at 65
msg 49 sent by (9, 8) at 60, delivered to (7, 1) at 73
msg 50 sent by (7, 5) at 61, delivered to (9, 3) at 72
msg 51 sent by (9, 8) at 61, delivered to (8, 9) at 72
msg 52 sent by (6, 9) at 63, delivered to (4, 4) at 77
msg 53 sent by (6, 7) at 64, delivered to (8, 7) at 67
msg 54 sent by (7, 0) at 66, delivered to (8, 6) at 75
msg 55 sent by (0, 0) at 66, delivered to (6, 7) at 80
msg 56 sent by (4, 6) at 66, delivered to (3, 5) at 85
msg 57 sent by (7, 6) at 67, delivered to (3, 1) at 79
msg 58 sent by (2, 7) at 69, delivered to (7, 2) at 80
msg 59 sent by (9, 5) at 69, delivered to (4, 1) at 81
msg 60 sent by (2, 3) at 71, delivered to (4, 2) at 83
msg 61 sent by (7, 7) at 72, delivered to (9, 0) at 78
msg 62 sent by (7, 2) at 73, delivered to (5, 9) at 89
msg 63 sent by (5, 0) at 74, delivered to (3, 5) at 88
msg 64 sent by (7, 1) at 74, delivered to (1, 8) at 86
msg 65 sent by (1, 6) at 76, delivered to (9, 5) at 95
msg 66 sent by (0, 5) at 78, delivered to (6, 9) at 89
msg 67 sent by (9, 7) at 79, delivered to (6, 3) at 93
msg 68 sent by (3, 9) at 81, delivered to (4, 2) at 86
msg 69 sent by (9, 6) at 82, delivered to (0, 3) at 91
msg 70 sent by (0, 3) at 82, delivered to (3, 3) at 86
msg 71 sent by (7, 0) at 84, delivered to (1, 4) at 93
msg 72 sent by (2, 4) at 85, delivered to (0, 1) at 101
msg 73 sent by (6, 1) at 86, delivered to (3, 2) at 95
msg 74 sent by (0, 8) at 88, delivered to (7, 1) at 99
msg 75 sent by (5, 2) at 88, delivered to (1, 7) at 100
msg 76 sent by (0, 8) at 88, delivered to (5, 9) at 96
msg 77 sent by (3, 2) at 90, delivered to (6, 4) at 96
msg 78 sent by (9, 8) at 91, delivered to (1, 7) at 103
msg 79 sent by (9, 3) at 92, delivered to (5, 6) at 102
msg 80 sent by (3, 4) at 92, delivered to (5, 1) at 102
msg 81 sent by (6, 9) at 92, delivered to (7, 5) at 100
msg 82 sent by (2, 4) at 93, delivered to (1, 8) at 107
msg 83 sent by (3, 8) at 94, delivered to (3, 6) at 103
msg 84 sent by (4, 1) at 94, delivered to (2, 4) at 106
msg 85 sent by (1, 1) at 95, delivered to (5, 0) at 109
msg 86 sent by (0, 4) at 96, delivered to (7, 4) at 104
msg 87 sent by (4, 7) at 96, delivered to (6, 3) at 105
msg 88 sent by (5, 1) at 97, delivered to (4, 5) at 111
msg 89 sent by (3, 4) at 98, delivered to (6, 2) at 110
msg 90 sent by (4, 0) at 100, delivered to (1, 0) at 108
msg 91 sent by (4, 4) at 100, delivered to (1, 9) at 113
msg 92 sent by (6, 6) at 101, delivered to (3, 2) at 115
msg 93 sent by (4, 3) at 102, delivered to (4, 7) at 107
msg 94 sent by (4, 5) at 103, delivered to (3, 1) at 119
msg 95 sent by (0, 4) at 103, delivered to (6, 2) at 119
msg 96 sent by (3, 2) at 104, delivered to (2, 1) at 123
msg 97 sent by (9, 8) at 105, delivered to (5, 7) at 121
msg 98 sent by (0, 6) at 105, delivered to (2, 5) at 119
msg 99 sent by (4, 4) at 107, delivered to (4, 4) at 108
msg 100 sent by (2, 8) at 107, delivered to (0, 5) at 123
